src/controllers/sync-controller.ts

```diff
@@ -143,10 +361,67 @@ export class SyncController implements ReactiveController {
     }
   }
 
+  /**
+   * Register periodic background sync if capabilities are available and settings allow
+   * Phase 1.3: App initialization periodic sync registration
+   */
+  async #registerPeriodicBackgroundSyncIfAvailable(): Promise<void> {
+    // Only proceed if we have both service worker and periodic background sync APIs
+    if (!this.#serviceWorkerReady || !this.#pwaCapabilities.periodicBackgroundSync) {
+      if (this.#serviceWorkerReady) {
+        DebugService.logInfo('sync', 'Periodic background sync API not available - falling back to timer-based sync when app is visible');
+      }
+      return;
+    }
+
+    try {
+      // Wait for settings to be available
+      if (this.isSettingsLoading) {
+        // Listen for settings to become available
+        const settings = await new Promise<AppSettings | null>((resolve) => {
+          const checkSettings = () => {
+            if (!this.isSettingsLoading) {
+              resolve(this.settings);
+            } else {
+              setTimeout(checkSettings, 100);
+            }
+          };
+          checkSettings();
+        });
+
+        if (!settings) {
+          DebugService.logInfo('sync', 'No settings available - periodic background sync not registered');
+          return;
+        }
+      }
+
+      const settings = this.settings;
+      if (!settings || !settings.auto_sync) {
+        DebugService.logInfo('sync', 'Auto-sync not enabled in settings - periodic background sync not registered');
+        return;
+      }
+
+      // TODO: Add user permission and PWA installation checks
+      // For now, register if API is available and auto-sync is enabled
+
+      // Send message to service worker to register periodic background sync
+      // Note: Sync interval will be handled by service worker default (30 minutes)
+      if (navigator.serviceWorker.controller) {
+        navigator.serviceWorker.controller.postMessage({
+          type: 'REGISTER_PERIODIC_SYNC'
+        });
+        DebugService.logInfo('sync', 'Periodic background sync registration requested');
+      }
+    } catch (error) {
+      DebugService.logWarning('sync', 'Failed to register periodic background sync', { error });
+    }
+  }
+
+
   private handleServiceWorkerMessage(message: SyncMessage) {
     switch (message.type) {
       case 'SW_LOG':
```

Comment:
The service worker does various non-sync stuff too. Would it make more sense for the DebugService to connect to the service worker directly to collect log messages? We want the log messages to go to the Debug Service even if we're on a screen that doesn't use the SyncController.